### ISSUE TITLE
Link is not clearly identifiable: color contrast for MCO link on home page

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5579]
Compliance level: A

Pages impacted: MCO home page, link to MCO wiki in gray box

In MCO, color is the only way to distinguish links from other text. W3C would prefer that you add another way to distinguish links (we use underline on hover). The message for this error is not particularly helpful, as this link is not handled differently than other links in MCO.

According to Adam and Randall, if color is the only visual cue to identify a link (i.e., they are't underlined, bolded, etc.), then it needs to have a 3:1 color contrast with surrounding text (or, in this case, presumably the gray background). 

We may be able to resolve this by making the gray background of the box lighter. We could probably also solve it by having all links in MCO underlined by default (this is how IUCAT handles links), but that change would be larger in scope. Right now, we are only getting an error for this specific link.

<img width="1212" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOUxQQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9bc3dfb8bcd591fbe795b12188bedc4b61e1c4f5/image.png" alt="image.png" />

# Done Looks Like
- [x] Gray background removed from element
- [x] Add 5px to all sides of box-shadow on #home-visit-box to create a solid outline

# Mockup
<img width="805" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1RWQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--93148ff6c03a6ee45ba6d98a1af2861f01cee5bd/Screenshot%202024-01-17%20at%202.32.19%E2%80%AFPM.png" alt="Screenshot 2024-01-17 at 2.32.19 PM.png" />

**Comment by cjcolvar** [comment]
Pushed to mco-staging for testing.

**Comment by elynema** [comment]
@cjcolvar @joncameron Just checked this with the SiteImprove browser plugin, and I'm still getting the same error: Links are not clearly identifiable. 

I did check and the text now has an 8:1 contrast ratio, so that should be fine. 

<img width="1300" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNGdMQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--daf158c0392e7568174a00f08b03f92646304d43/image.png" alt="image.png" />

The information in SiteImprove is basically telling us to add an underline on focus/hover, which we already do. I'm wondering if something about that box is making it not recognize that CSS in this single scenario. Otherwise, I really don't know how to better solve this error.

**Comment by elynema** [comment]
@cjcolvar First question: does this element have underline on both focus and hover? It wasn't clear to me from trying to inspect the CSS. Here's what SiteImprove recommends:
   ```
a:hover,
    a:focus {
        text-decoration: underline;
    }
```

It looks to me like we have hover, but when I try to tab through the page, there is no indication when this link is focused.

**Comment by elynema** [comment]
If that doesn't work, Richard suggested that links that are within <p> have to be identifiable by more than just color (this is not a requirement for navigational links where page layout makes their identity as a link obvious). This would require this link to be always underlined.

Here is the reference on that: https://www.w3.org/WAI/WCAG21/Techniques/failures/F73

This may be what SiteImprove is trying to say, but their code snippet is not quite correct. SiteImprove says: "Color alone is not enough to distinguish a link from the surrounding text."

It's possible that SiteImprove is missing other links within text on the MCO homepage b/c they are not within <p> elements.

**Comment by cjcolvar** [comment]
I'll add the underline on hover and see if that clears it up.  When I did it within the browser dev tools before it was still complaining about a.link but I'm not really sure what is because there isn't a JS `link` event as far as I know.

**Comment by cjcolvar** [comment]
I found out that `link` is the link state when no events/states are set...so when seen as-is.  It looks like Richard's advice is needed to handle this case.  Taking out the `<p>` messes with the CSS.  I made a few quick changes and got something that I think looks decent.  I could try to replicate the previous look if it is important.

Before:
![Screenshot from 2024-02-02 13-48-37](https://github.com/avalonmediasystem/avalon/assets/1053603/77abf2be-13c4-4dbb-bdd5-3fba340425ff)

After:
![Screenshot from 2024-02-02 13-49-03](https://github.com/avalonmediasystem/avalon/assets/1053603/08dcf14f-2242-4257-9bda-69286a0e4bbd)

**Comment by elynema** [comment]
@cjcolvar This looks like a working temp fix and solves the error reported by SiteImprove. We should probably have a broader conversation later about underlining link text.
